src/Components/Dashboard/Dashboard.tsx

```diff
@@ -19,5 +57,38 @@ export default function Dashboard({userData}:any) {
    
     </div>
    </div>
-  )
+        <div className="row">
+          <div className="col-md-6">
+            <div>
+              <h3 className="pt-5 mt-5">Tasks</h3>
+              <span>Lorem ipsum dolor sit amet,consecteture</span>
+              <div className="row">
+                <div className="col-md-4 d-flex p-4 m-4 ">
+                  <div className=" rounded-3 colorr-one p-3 m-2 text-center" >
+                    <img src={icon} alt="#" className="rounded-2 color-one "  style={{ width: '30px' }} />
+                    <div className="text-#6F7881 " >Todo</div>
+                    <h6>{toDoCount}</h6>
+                  </div>
+                  <div className=" rounded-3 colorr-two p-3 m-2 text-center" >
+                    <img src={iconn} alt="#" className="rounded-2 color-two"  style={{ width: '30px' }} />
+                    <div className="text-#6F7881 " >Inprogress</div>
+                    <h6>{progressCount}</h6>
+                  </div>
+                  <div className=" rounded-3 colorr-three p-3 m-2 text-center" >
+                    <img src={iconnn} alt="#" className="rounded-2 color-three"  style={{ width: '30px' }} />
+                    <div className="text-#6F7881 " >Done</div>
+                    <h6>{doneCount}</h6>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+          <div className="col-md-6">
```

Comment:
Users charts should be rendered conditionally. If the user is a manager, we show it, otherwise, we should remove it from the dashboard.